tests/e2e/utils/infrastructure.py

```diff
@@ -177,9 +177,9 @@ def deploy(self) -> Dict[str, str]:
             CloudFormation Stack Outputs with output key and value
         """
         stack_file = self._create_temp_cdk_app()
-        synth_command = f"npx cdk synth --app 'python {stack_file}' -o {self._cdk_out_dir}"
+        synth_command = f"npx cdk@latest synth --app 'python {stack_file}' -o {self._cdk_out_dir}"
         deploy_command = (
-            f"npx cdk deploy --app '{self._cdk_out_dir}' -O {self._stack_outputs_file} "
+            f"npx cdk@latest deploy --app '{self._cdk_out_dir}' -O {self._stack_outputs_file} "
```

Comment:
This is not a good idea as it will create mismatches with the pinned version of CDK in our deps. We rely on dependabot to update the CDK version everywhere (JS and Python).

Reply:
You're 100% right, Ruben! It was a leftover from my local tests.
Removed